.github/workflows/workflow.yml

```diff
@@ -1,57 +1,85 @@
 name: Main workflow
 on: [push, pull_request]
 jobs:
-  run:
-    name: Run
+  build:
     runs-on: ${{ matrix.operating-system }}
     strategy:
       matrix:
         operating-system: [ubuntu-latest, windows-latest]
     steps:
     - name: Checkout
       uses: actions/checkout@v2
-
-    - name: Setup Node.js 12.x
+    - name: Setup Node.js 12
```

Comment:
i removed the empty lines between steps since the file is much larger now (tighten up individual jobs, to make file more skimmable)